boards/waveshare-esp32-s3-zero.json

```diff
@@ -0,0 +1,47 @@
+{
+  "build": {
+    "arduino": {
+      "ldscript": "esp32s3_out.ld",
+      "memory_type": "qio_qspi"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DBOARD_HAS_PSRAM",
+      "-DARDUINO_ESP32_S3_ZERO",
```

Comment:
Shouldn't it be `-DARDUINO_WAVESHARE_ESP32_S3_ZERO`?

The same applies for the boards below.